pkg/metrics/stats_reporter.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2020 The Knative Authors
+Copyright 2024 The Knative Authors
```

Comment:
```suggestion
Copyright 2020 The Knative Authors
```
nit: don't change the license year